src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -359,4 +370,51 @@ private function getContainerEnvVars(ContainerBuilder $container): array
 
         return array_values($envs);
     }
+
+    protected function getUsagesForDefinition(Definition $definition)
+    {
+        $class = $definition->getClass();
+
+        return null !== $class ? $this->getUsages($class) : [];
+    }
+
+    protected function getUsages(string $class)
+    {
+        return \array_key_exists($class, $this->usages) ? $this->usages[$class] : [];
+    }
+
+    protected function buildUsageMapForContainer(ContainerBuilder $builder)
```

Comment:
```suggestion
    private function buildUsageMapForContainer(ContainerBuilder $builder)
```

I don't think that the descriptor implementations need to override/call this method, so I would set its visibility to `private`. Same for `buildUsageMapForDefinition` and `buildUsageMapForArgument`.

Reply:
Fixed!